e2e/assets/drift/correction-disabled/gitrepo.yaml

```diff
@@ -3,8 +3,8 @@ apiVersion: fleet.cattle.io/v1alpha1
 metadata:
   name: drift-test
 spec:
-  repo: https://github.com/rancher/fleet-test-data
-  branch: master
+  repo: https://github.com/weyfonk/fleet-test-data
+  branch: faster-drift-e2e-tests
```

Comment:
We should be using https://github.com/rancher/fleet-test-data now that https://github.com/rancher/fleet-test-data/pull/22 was merged, right?

Reply:
Oops, I had overlooked that. Of course we should. Fixed now, thanks :)